app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,7 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user_index = params[:user_id]
+  end
 
```

Comment:
- [ ] ApplicationController file `app/controllers/application_controller.rb` should contain `#current_user` method. Kindly improve upon this by adding code like below:
```ruby
def current_user
  User.first
end
```

Reply:
thanks a lot, I just forgot that point